mobile/src/libs/components/player/components/progress-bar/styles.ts

```diff
@@ -14,16 +14,32 @@ const styles = StyleSheet.create({
     bottom: 20,
     fontFamily: FontFamily.MONTSERRAT_400,
     fontSize: 12,
-    color: AppColor.GRAY_400,
   },
   duration: {
     position: 'absolute',
     right: 17,
     bottom: 20,
     fontFamily: FontFamily.MONTSERRAT_400,
     fontSize: 12,
+  },
+});
+
+const lightStyles = StyleSheet.create({
+  progress: {
+    color: AppColor.GRAY_400,
+  },
+  duration: {
+    color: AppColor.GRAY_400,
+  },
+});
+
+const darkStyles = StyleSheet.create({
+  progress: {
+    color: AppColor.GRAY_400,
+  },
+  duration: {
     color: AppColor.GRAY_400,
   },
 });
```

Comment:
Why do we need two identical styles?

Reply:
@danilittaa did you read that i have written in description ?)

Reply:
Yes, did you read my comment? 
Why do we need two absolutely identical styles? (I would understand if you had used different colors, but you made everything the same)

Reply:
Arent we suppose to have the same colors ? If you think its not necessary please take android with already merged prs and check how does it work now with just one color setted.

Reply:
For now, there's no point in creating two separate styles since the color scheme for the light and dark themes is the same. You can simply reference the same file

Reply:
do you think it looks better and person who is not familiar with code will look at it and wont think that it not makes any lexical sence ?
![Снимок экрана 2023-09-28 113025](https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/assets/111873795/2f2f772a-951d-4d1a-9bf6-611f00f63929)

Reply:
I think we don't need this line at all. It seems to me (but I'm not sure) that the color issue in dark mode should have been resolved by simply adding color. That's why Lisa in the chat [here](https://bsa2023.slack.com/archives/C05P563E2MN/p1695832484447239?thread_ts=1695826530.921589&cid=C05P563E2MN) said that QA should check it first. Did you ask QA to verify it?

Reply:
I checked it in the morning form real device, there was this problem.